recipes/cppbenchmark/all/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.13)
+project(cmake_wrapper)
+
+include(${CMAKE_CURRENT_BINARY_DIR}/../conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(WIN32)
+    add_definitions (-D_CRT_SECURE_NO_WARNINGS)
+endif()
+
+enable_testing()
+
+add_subdirectory("source_subfolder")
+
+conan_target_link_libraries(cppbenchmark)
+
+target_compile_features(cppbenchmark PUBLIC cxx_std_17)
+if(CPPBENCHMARK_TESTS)
```

Comment:
testing should not be enabled in CCI. you can remove it.

Reply:
fixed in 29610bc0